src/components/dashboard/OrdersChart.tsx

```
@@ -0,0 +1,444 @@
+
+/**
+ * Orders chart component displaying daily order counts as bars
+ * with optional color coding based on performance thresholds
+ */
+export function OrdersChart({
+  data = [],
+  loading = false,
+  error,
+  height = 400,
+  locale = 'pt-BR',
+  timezone = 'America/Sao_Paulo',
```

ESLint

[error] 'timezone' is assigned a value but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)